docs/Integrations/payments/revenuecat.md

```diff
@@ -0,0 +1,238 @@
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+## 4. Add RevenueCat actions
+
+To manage in-app purchases and subscriptions inside your FlutterFlow app, you have to use the RevenueCat Actions. Below are the types of RevenueCat action:
+
+- **Paywall**
+- **Purchase**
+- **Restore Purchases**
+
+### Paywall [Action]
+
+This action checks whether a user has purchased an item. If not, you can open the Paywall (asking to buy an item or purchase a subscription).
+
+Follow the steps below to see if a user is subscribed and take action accordingly.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/0m0oeM9PRbe7e5JbapCf?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Purchase [Action]
+
+This action allows you to purchase the item. Here’s how you add it:
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/95HnVG5vM9J7gi73vjxW?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Restore Purchases [Action]
+
+Using this action, you can allow users to re-activate the subscription he has already paid for. This is helpful when a user has reinstalled the app or logged in to a new device.
+
+:::info
+
+- A good practice is to allow users to manually restore the purchase by showing a button or text (maybe on a paywall/settings page).
+- If you provide this option, please check [**How RevenueCat should respond to restore behavior**](https://www.revenuecat.com/docs/restoring-purchases#restore-behavior).
+
+:::
+
+![adding-restore-purchase-action.avif](adding-restore-purchase-action.avif)
```

Comment:
Add a figure tag and write a small caption for the image

Reply:
Done!